extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -272,17 +272,11 @@ export class ProjectsController {
 		// check if schema compare extension is installed
 		if (vscode.extensions.getExtension(constants.schemaCompareExtensionId)) {
 			// build project
-			await this.buildProject(treeNode);
-
-			// start schema compare with the dacpac produced from build
-			const project = this.getProjectFromContext(treeNode);
-			const dacpacPath = path.join(project.projectFolderPath, 'bin', 'Debug', `${project.projectFileName}.dacpac`);
+			const dacpacPath = await this.buildProject(treeNode);
 
 			// check that dacpac exists
 			if (await utils.exists(dacpacPath)) {
```

Comment:
we should still give the message that dacpac wasnt created and SC cant start... right?

Reply:
updated the error message to be because the build failed(a dacpac from a previous successful build could still exist)